workflow/artifacts/gcs/gcs.go

```diff
@@ -249,3 +253,7 @@ func uploadObject(client *storage.Client, bucket, key, localPath string) error {
 	}
 	return nil
 }
+
+func (g *ArtifactDriver) ListObjects(artifact *wfv1.Artifact) ([]string, error) {
+	panic("implement me")
```

Comment:
I actually think we should return an error here (rather than panic) - and do the implementation later